### ISSUE TITLE
Featured collections are not displayed for anonymous users

### ISSUE THREAD

**Issue by birkland** [jhu-idc/iDC-general#433]
To reproduce:
* Log in as admin.  Create a collection whose metadata has the "featured item" checkbox checked.
* Go to the [collections page](https://islandora-idc.traefik.me/collections) logged in as admin.  You'll see a "Featured collections" section with your collection
* log out, and navigate back to the [collections page](https://islandora-idc.traefik.me/collections).  You will not see any featured collections.

What you see as admin:
![image](https://user-images.githubusercontent.com/517881/136272806-e9a0e015-a565-41f3-b489-add3ff4c859f.png)

What you see as anonymous:
![image](https://user-images.githubusercontent.com/517881/136272890-e3b65c2f-c4cb-4052-a4aa-ae340bde0272.png)

**Comment by jabrah** [comment]
Looks like it might somehow be a caching issue. Running through the above steps demonstrates the issue, but if you clear Drupal's cache (I ran `make cache-rebuild` on the command line, so I wouldn't have to re-login as admin) and refresh the new Featured Item appears.

**Comment by jabrah** [comment]
Looks like we can completely disable caching on the `/collections` route by using a `no_cache` option in the route configuration. Local testing, at least, looked promising. Not sure what sort of performance impact this may have


`idc_ui_module.routing.yml`
```
idc-ui-module.collections:
  path: 'collections'
  defaults:
    _controller: '\Drupal\idc_ui_module\Controller\CollectionsController::collections'
    _title: 'All collections'
  options:
    no_cache: TRUE
  requirements:
    _permission: 'access content'
```

Sort of interesting, but we actually set the cache max-age to 0 in the Drupal controller for this route. However, it seems like Drupal happily ignores this for anonymous users (if the way its set in the controller even does anything)

**Comment by jabrah** [comment]
Note, if the above patch is not sufficient, and/or if this issue appears on other pages, we might look at disabling the **Internal Page Cache** module in Drupal

**Comment by birkland** [comment]
Resolved in jhu-idc/idc-isle-dc#264
